Sprint-1/destructuring/exercise-3/exercise.js

```diff
@@ -6,3 +6,19 @@ let order = [
   { itemName: "Hot Coffee", quantity: 2, unitPricePence: 100 },
   { itemName: "Hash Brown", quantity: 4, unitPricePence: 40 },
 ];
+function receipt(arr){
+  let total=0;
+  const firstLine="QTY".padEnd(8)+"ITEM".padEnd(20)+"Total";
+  console.log(firstLine);
+
+
+arr.forEach(({itemName,quantity,unitPricePence})=> {
+  itemPrice=unitPricePence*quantity/100;
+  const line=String(quantity).padEnd(8)+itemName.padEnd(20)+itemPrice.toFixed(2);
```

Comment:
This has quite a lot of duplication with line 11. Imagine if you wanted to change how much padding there was between each column, you'd need to change both lines. Can you work out how to simplify this and avoid this problem?